denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc

```diff
@@ -32,18 +32,15 @@ tutorial_data:
     particle_guide: true
   4:
     hologram:
-      - "<&6><&l>Warp Command"
-      - "<&b>Warps can be made by any player within their claim."
-      - "<&e>You can visit server warps, or other player's warps."
-      - "<&a>Command Menu<&co> <&b>/warps"
+      - "<&6><&l>Vote Crates"
+      - "<&e>Voting for our server will earn you rewards."
+      - "<&e>use <&b>/vote <&e>to find out how."
```

Comment:
This line could be more consistent with capitalization.

Reply:
Done.